test/integration/imageimporter_test.go

```diff
@@ -993,49 +988,49 @@ func TestImageStreamImportRedHatRegistry(t *testing.T) {
 
 const etcdDigest = "sha256:958608f8ecc1dc62c93b6c610f3a834dae4220c9642e6e8b4e0f2b3ad7cbd238"
 const etcdManifest = `{
-   "schemaVersion": 1, 
```

Comment:
Some of the failures are caused by touching this part of the file. This is caused by digest not matching the version you've modified. No, I have no idea how to re-calculate those digests :/ 
Personally, I'm always failing back to VIM where I don't have auto-formatting, or any other auto-* scripts which will mess this part.